x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/request/RequestUtils.java

```diff
@@ -34,5 +34,9 @@ public static URI buildUri(URI accountUri, String service, CheckedSupplier<URI,
         }
     }
 
+    public static URI buildUri(String service, CheckedSupplier<URI, URISyntaxException> uriBuilder) {
+        return buildUri(null, service, uriBuilder);
```

Comment:
Just a helper that converts a URI exception to an `ElasticsearchStatusException`.